public/src/index.tsx

```diff
@@ -314,7 +313,9 @@ async function main(): Promise<void> {
 
     if (curRunning === DEBUG_SERVER) {
         const httpHelper = new HttpHelper("/api/v1");
-        const grpcHelper = new GrpcManager();
+        // TODO: use process.env.QUICKFEED_PORT
+        // https://webpack.js.org/plugins/environment-plugin/
+        const grpcHelper = new GrpcManager("https://" + window.location.hostname);
```

Comment:
Can this TODO be resolved easily, or will this require much work?? Also, check out what @nicro950 did in #493 regarding this stuff, to ensure things are compatible. Note that, I think `"https://"` can be replaced with `window.location.protocol`...

PS: @JosteinLindhom managed to remove these arguments to `GrpcManager`, but I'm clueless about this stuff...

Reply:
Yes, I let it as TODO because I was think to make a separate PR to also adjust this `DEBUG_SERVER` and `DEBUG_BROWSER` flags.